lib/spack/docs/tutorial_modules.rst

```diff
@@ -58,42 +284,78 @@ something like this:
   -- gcc ----------------------------------------------------------
   gcc@4.8
 
-In order to showcase the capabilities of module customization, we will want to
-build a limited set of packages with multiple compilers. If you do not already
-have multiple compilers listed by ``spack compilers``, you should build one
-with Spack:
+Let's bootstrap a more recent compiler with the one that was detected automatically:
+
+.. code-block:: console
+
+  $ spack install gcc@7.2.0
+  ...
+  Wait a long time
```

Comment:
But like, seriously. Takes an hour on a laptop. Not feasible for an in-person tutorial, but is nice for an "on your own time" tutorial

Reply:
I wrote this tutorial thinking at a stand-alone piece of documentation, unrelated to the context of SC. The idea is that we can reuse this for an in-person tutorial, not *use this just for it*.